src/guide/code-owners.md

```diff
@@ -27,182 +26,189 @@ This list is intended to make it easier to identify which email list to include
+      * [org/objectweb](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/org/objectweb) - Core Libs
+      * [org/xml](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/org/xml) - Core Libs
+      * [perf](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/perf) - Runtime
+      * [[linux](https://github.com/openjdk/jdk/tree/master/java.base/linux/classes/jdk/internal/platform), [share](https://github.com/openjdk/jdk/tree/master/java.base/share/classes/jdk/internal/platform), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/classes/jdk/internal/platform), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/classes/jdk/internal/platform)]/platform - HotSpot
+      * [ref](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/ref) - Core Libs, GC
+      * [reflect](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/reflect) - Core Libs
+      * [util/random](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/util/random) - Core Libs
+      * [util/regex](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/util/regex) - Core Libs
+      * [util/xml](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/util/xml) - Core Libs
+      * [vm](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/jdk/internal/vm) - HotSpot
+    * [invoke](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/sun/invoke) - Core Libs
+    * [[share](https://github.com/openjdk/jdk/tree/master/java.base/share/classes/java/io), [sun](https://github.com/openjdk/jdk/tree/master/java.base/windows/classes/sun/io), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/classes/java/io)]/io - NIO
+    * [[macosx](https://github.com/openjdk/jdk/tree/master/java.base/macosx/classes/java), [share](https://github.com/openjdk/jdk/tree/master/java.base/share/classes/java), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/classes/java), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/classes/java)]/java
+      * [[share](https://github.com/openjdk/jdk/tree/master/java.base/share/classes/java/lang), [unix](https://github.com/openjdk/jdk/tree/master/java.base/unix/classes/java/lang), [windows](https://github.com/openjdk/jdk/tree/master/java.base/windows/classes/java/lang)]/lang - Core Libs
+      * [math](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/java/math) - Core Libs
+      * [time](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/java/time) - Core Libs
+    * [launcher](https://github.com/openjdk/jdk/tree/master/src/java.base/share/classes/sun/launcher) - Tools
```

Comment:
That may be where the historical ownership was but any changes to the launcher tend to be discussed/reviews on core-libs so probably better to change this owner.

Reply:
Added Core-Libs.